torchvision/models/detection/_utils.py

```diff
@@ -346,3 +346,16 @@ def set_low_quality_matches_(self, matches, all_matches, match_quality_matrix):
 
         pred_inds_to_update = gt_pred_pairs_of_highest_quality[:, 1]
         matches[pred_inds_to_update] = all_matches[pred_inds_to_update]
+
+
+def smooth_l1_loss(input, target, beta=1. / 9, size_average=True):
```

Comment:
Can you add type annotations for `beta` and `size_average` using type annotations from Python 3?